### PR TITLE
fix: add yq package

### PR DESCRIPTION
**Pull request by kameshsampath**
Add [yq](https://github.com/mikefarah/yq) package 

#542

### CODE REVIEW

File: yq.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  packages:
+    - busybox
+    - go
+pipeline:
+  - uses: go/install
+    with:
+      package: github.com/mikefarah/yq/v4
+      version: v${{package.version}}
+  - runs: |
+      ${{targets.destdir}}/usr/bin/yq --version | grep "version v${{package.version}}"
```
Comment (tuananh) [inline yq.yaml:26]: testing?

Comment (kameshsampath) [reply yq.yaml:26]: yeah, just want to make sure the right version of package is installed.

@tuananh ^^

**Comment by tuananh** [comment]
@rawlingsj `go/install` support installing from a branch, `v4` in this case but it's kind of a moving target. the version doesn't quite make sense in this case.

IMO, the version should point toward a specific, non-moving target (specific commit hash, a specific file with static hash, etc..)

wdyt?

**Comment by rawlingsj** [comment]
yeah agreed - we only tend to use tags where they are fixed as that can lead to un expected versions

**Comment by kameshsampath** [comment]
@tuananh  and @rawlingsj  

In this case I am installing yq v4.30.8 --> https://github.com/mikefarah/yq/blob/v4.30.8/go.mod and the module name is github.com/mikefarah/yq/v4, so I used as package name and version `${{package.version}}`.   so ideally it will be translated to `go install github.com/mikefarah/yq/v4@v4.30.8` 

Is that something not right? Sorry I did not understand the issue here.

**Comment by tuananh** [comment]
> @tuananh and @rawlingsj
> 
> In this case I am installing yq v4.30.8 --> https://github.com/mikefarah/yq/blob/v4.30.8/go.mod and the module name is github.com/mikefarah/yq/v4, so I used as package name and version `${{package.version}}`. so ideally it will be translated to `go install github.com/mikefarah/yq/v4@v4.30.8`
> 
> Is that something not right? Sorry I did not understand the issue here.

my bad. i thought version is 4. sorry

**Review by tuananh — COMMENTED** [review]
lgtm

**Comment by kameshsampath** [comment]
> > @tuananh and @rawlingsj
> > In this case I am installing yq v4.30.8 --> https://github.com/mikefarah/yq/blob/v4.30.8/go.mod and the module name is github.com/mikefarah/yq/v4, so I used as package name and version `${{package.version}}`. so ideally it will be translated to `go install github.com/mikefarah/yq/v4@v4.30.8`
> > Is that something not right? Sorry I did not understand the issue here.
> 
> my bad. i thought version is 4. sorry

no worries.

**Review by imjasonh — DISMISSED** [review]

**Review by imjasonh — APPROVED** [review]

**Comment by eddiezane** [comment]
Hey @kameshsampath,

Thanks for this contribution! I'd like to send some swag your way. If interested could you please shoot an email to eddiezane at chainguard dot dev?

**Pull request merged**